ros_ws/src/teleoperation/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.3)
 project(teleoperation)
 
 ## Compile as C++11, supported in ROS Kinetic and newer
-set (CMAKE_CXX_STANDARD 11)
+add_compile_options(-std=c++11)
```

Comment:
Why did you change this? Setting the `CMAKE_CXX_STANDARD` should be prefered over setting the Argument manually.

Reply:
because std::unique_ptr